tests/helpers/mod.rs

```diff
@@ -140,3 +140,52 @@ macro_rules! test_lint_diagnostics {
       ::insta::assert_snapshot!(formatted_diags, @$expected_diagnostics);
   }};
 }
+
+use cairo_lang_filesystem::db::CrateSettings;
+use cairo_lang_utils::Intern;
+
+use crate::CRATE_CONFIG;
+
+pub fn setup_test_crate_ex(db: &mut dyn SemanticGroup, content: &str) -> CrateId {
+    let file_id = FileLongId::Virtual(VirtualFile {
+        parent: None,
+        name: "lib.cairo".into(),
+        content: content.into(),
+        code_mappings: [].into(),
+        kind: FileKind::Module,
+    })
+    .intern(db);
+
+    let settings = CrateSettings {
+        name: None,
+        edition: Edition::latest(),
+        version: None,
+        dependencies: Default::default(),
+        experimental_features: ExperimentalFeaturesConfig {
+            negative_impls: true,
+            associated_item_constraints: true,
+            coupons: true,
+        },
+        cfg_set: Default::default(),
+    };
+    let crate_config = CrateConfiguration {
+        root: Directory::Virtual {
+            files: BTreeMap::from([("lib.cairo".into(), file_id)]),
+            dirs: Default::default(),
+        },
+        settings,
+        cache_file: None,
+    };
+
+    let crate_id = CrateLongId::Virtual {
+        name: "test".into(),
+        file_id,
+        settings: CRATE_CONFIG.to_string(),
+        cache_file: None,
+    }
+    .intern(db);
+    // TODO (wawel37) make it in proper way
```

Comment:
I think we can get rid of this TODO, as tests work as expected (relating to compiler generated code)